app/views/no_businesses/new.html.haml

```diff
@@ -1,13 +1,13 @@
-%h1.heading-xlarge
-  Report no business on
-  = @task.framework.short_name
+.govuk-grid-row
+  .govuk-grid-column-full
+    %h1.govuk-heading-xl
+      Report no business
 
-%p
-  By reporting no business you are confirming that you have
-  no transactions to report on this framework for
-  = task_period_date(@task)
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    %p
+      By reporting no business you are confirming that you have no transactions
```

Comment:
Again, I'm not convinced that removing indicators of the framework/period of submission is a good idea.

Reply:
See above.